native/c/zowex.cpp

```diff
@@ -789,6 +802,44 @@ int handle_job_submit(ZCLIResult result)
   return RTNCD_SUCCESS;
 }
 
+int handle_job_submit_uss(ZCLIResult result)
+{
+  int rc = 0;
+  ZJB zjb = {0};
+  string file(result.get_positional("file-path")->get_value());
+
+  ZUSF zusf = {0};
+  string response;
+  rc = zusf_read_from_uss_file(&zusf, file, response);
+  if (0 != rc)
+  {
+    cerr << "Error: could not view USS file: '" << file << "' rc: '" << rc << "'" << endl;
+    cerr << "  Details:\n"
+         << zusf.diag.e_msg << endl
+         << response << endl;
+    return RTNCD_FAILURE;
+  }
+
+  vector<ZJob> jobs;
+  string jobid;
+  rc = zjb_submit(&zjb, response, jobid);
+
+  if (0 != rc)
+  {
+    cerr << "Error: could not submit JCL: '" << file << "' rc: '" << rc << "'" << endl;
+    cerr << "  Details: " << zjb.diag.e_msg << endl;
+    return RTNCD_FAILURE;
+  }
+
+  string only_jobid(result.get_option("--only-jobid")->get_value());
+  if ("true" == only_jobid)
+    cout << jobid << endl;
```

Comment:
Now that I'm reading the handler logic, can we remove the trim from the Go code since we don't print any spaces?